ui/src/scenes/Projects/components/ProjectCreate/index.js

```diff
@@ -31,7 +37,40 @@ class ProjectCreate extends Component {
       }
     );
   };
+ 
+  //Called on change of any field in form
+  onFormChange(e) {
+
+    //Setting the required states with change of form field
+    this.setState({
+      [e.target.name]: e.target.value
+    },function() {
 
+      //Checking the fields to be null
+      if( this.state.name && 
+        this.state.description && 
```

Comment:
please refactor.

this.enable = isFormValid(this.state);

function isFormValid(state) {
  if isEmpty(state.name) return false;
  if isEmpty(state.description) return false;
  ...
  ...
}